compiler/src/iree/compiler/InputConversion/MHLO/MHLOToMHLOPreprocessing.cpp

```diff
@@ -526,6 +526,11 @@ struct ScatterOpImplicitBatch : public OpRewritePattern<mhlo::ScatterOp> {
           op, "Unable to add implicit batch dim to indice.");
     }
 
+    llvm::SmallVector<int64_t> newUpdateWindowDims;
+    for (auto dim : dimNumbers.getUpdateWindowDims()) {
+      newUpdateWindowDims.push_back(dim + 1);
+    }
```

Comment:
Why is it always +1? (I would have guessed it depends on where insertion happened)

Reply:
Added a comment. The batch dimension is inserted at the first index so everything shifts forward by 1.